### PR TITLE
docs: add example and equations to submit-data.md

### LINKED ISSUE #1021: add example to submit-data.md for gas calculations
https://github.com/celestiaorg/docs/pull/1009#issuecomment-1708945262

### PR DESCRIPTION
**Pull request by jcstein**
## Overview

Resolves #1021 

screenshot 2023-09-06 at 18 03 25

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

File: docs/concepts/submit-data.md [hunk 1]
```diff
@@ -58,6 +58,34 @@ The `GasCostPerBlobByte` and `GasCostPerTransactionByte` are parameters that
...
+willing to spend on a transaction. It is determined by both a static
+"fixed cost" and a dynamic cost based on the size of each blob involved
+in the transaction:
+
+$\text{Gas Limit} = \text{Fixed Cost} + \sum_{i=1}^{n} \text{Size of Blob}_i \times \text{Gas Cost Per Blob Byte}$
+
+Where:
+
+* $\text{Fixed Cost}$ is a static value (default is 65,000 gas)
+* $\sum_{i=1}^{n} \text{Size of Blob}_i$ is the size of the ith blob in the transaction
```
Comment (adlerjohn) [inline docs/concepts/submit-data.md:78]:
```suggestion
* $\sum_{i=1}^{n} \text{Size of Blob}_i$ is the size of the $i$th blob in the transaction
```

File: docs/concepts/submit-data.md [hunk 2]
```diff
@@ -58,6 +58,34 @@ The `GasCostPerBlobByte` and `GasCostPerTransactionByte` are parameters that
...
+* $\text{Gas Cost Per Blob Byte}$ is a parameter that could potentially be adjusted through the system's
+governance mechanisms.
+
+The gas price is set by the user when they submit a transaction. It is often
+specified by users directly. The total fee for the transaction is then
+calculated as the product of the estimated gas limit and the gas price.
+Since the state machine does not refund users for unused gas,
+it's important for users to estimate the gas limit accurately to
+avoid overpaying.
```
Comment (adlerjohn) [inline docs/concepts/submit-data.md:87]: Isn't it the case that users set the fee, not the gas price? Or did we change that?

Comment (jcstein) [reply docs/concepts/submit-data.md:87]: yep, thanks for catching that.

**Comment by jcstein** [comment]
on this episode of rebasing gone wrong

**Pull request closed without merging**